packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -41,12 +45,14 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
     try {
       const indyVdrPoolService = agentContext.dependencyManager.resolve(IndyVdrPoolService)
 
-      const did = didFromSchemaId(schemaId)
-
-      const pool = await indyVdrPoolService.getPoolForDid(agentContext, did)
-
+      // parse schema id (supports did:indy and legacy)
+      const { did, didIdentifier, schemaName, schemaVersion } = parseSchemaId(schemaId)
```

Comment:
This new `didIdentifier` term (used in various new RegExp) comes to replace legacy `unqualifiedDid`, right? As it corresponds to the namespace identifier (as defined in did:indy method spec for instance) maybe could it be called `namespaceIdentifier`?

Reply:
Yeah `namespaceIdentifier` is great!